remote_command_execution_vulnerability.py

```diff
@@ -32,6 +65,9 @@
 # proxies = {"http":"http://127.0.0.1:8080"}
 proxies = {}
 
+from tcp_file_server import TcpFileServer
+file_server = TcpFileServer("script_tools")

```

Comment:
Would it be possible to ask the user if he/she wants to use a tcp server?

Context: starting a tcp server is not 100% reliable, as it can fail because it depends on the network setup of the user. I believe most of the times is more reliable and easier to get the files from the internet as it is done now, the only scenario where this is not the case may be China. I would like to support both options

Reply:
I have added an option to download files from GitHub instead of a local TCP server.
BTW: I am in China, the GitHub access is not reliable here, so the local TCP server is necessary for me :)